vane-trifles/src/main/java/org/oddlama/vane/trifles/ChestSorter.java

```diff
@@ -166,42 +166,29 @@ public void on_player_right_click(final PlayerInteractEvent event) {
 		final var face = ((FaceAttachable) button_data).getAttachedFace();
 
 		int rx = 0;
-		int ry = 0;
+		int ry;
 		int rz = 0;
 
 		// Determine relative radius rx, ry, rz as seen from the button.
 		if (face == FaceAttachable.AttachedFace.WALL) {
 			ry = config_radius_y;
-			switch (facing) {
-				case NORTH:
-				case SOUTH:
-					rx = config_radius_x;
-					rz = config_radius_z;
-					break;
-				case EAST:
-				case WEST:
-					rx = config_radius_z;
-					rz = config_radius_x;
```

Comment:
Careful, this is different from the refactored code. X=Z here, but X=X below. That's different for all assignments. I think both switches must stay...

Reply:
Oh wow, good catch! I looked over this multiple times thinking that was going to be the case and never caught that :sweat_smile:.